mailbox/hbase/src/test/java/org/apache/james/mailbox/hbase/HBaseUtilsTest.java

```diff
@@ -147,7 +148,8 @@ public void testFlagsToPut() {
         flags.add("userFlag1");
         flags.add("userFlag2");
         HBaseId uuid = HBaseId.of(UUID.randomUUID());
-        final SimpleMailboxMessage message = new SimpleMailboxMessage(new Date(), 100, 10, null, flags, new PropertyBuilder(), uuid);
+        MessageId messageId = null;
+        final SimpleMailboxMessage message = new SimpleMailboxMessage(messageId, new Date(), 100, 10, null, flags, new PropertyBuilder(), uuid);
```

Comment:
It really means we have a design issue : most backend won't implement messageId, we should not need to pass null everywhere